src/pretix/base/forms/questions.py

```diff
@@ -162,12 +164,18 @@ def __init__(self, *args, **kwargs):
                     **d,
                     choices=[('', '')] + [(d, d) for d in self.scheme_titles[1]]
                 )
-                field.part_name = fname
-                fields.append(field)
+
+            elif fname == 'salutation':
+                d = dict(defaults)
+                d.pop('max_length', None)
+                field = forms.ChoiceField(
+                    **d,
+                    choices=[('', '---')] + [(s, s) for s in PERSON_NAME_SALUTATIONS]
```

Comment:
Is there a war to not duplicate the choices list creation?

Reply:
Not sure if worth it